docs/maps/mapml-viewer.md

```diff
@@ -0,0 +1,56 @@
+---
+id: mapml-viewer
+title: <mapml-viewer>
+slug: /maps/mapml-viewer
+---
+
+## Introduction
+
+The `<mapml-viewer>` element is the main element you can use to put a custom Web map on your page.  To create a (really) simple Web map, you might use it like this:
+
+```html
+<!DOCTYPE html>
+<html>
+   <head>
+     <title>A Simple Web Map[tm]</title>
+     <meta charset="utf-8" />
```

Comment:
`<meta charset="utf-8">` must exist in the first 1024 byte of a document, as such it is a best practice to specify it as the first element in `<head>`, and since people might copy and paste this we should adhere to that.

Reply:
I also recommend setting `<meta name="viewport" content="width=device-width, initial-scale=1">`, as almost everybody is going to want to build a responsive page.